test/spec/mox-spec.js

```diff
@@ -92,30 +99,42 @@ describe('The Mox library', function () {
 
     it('should mock a the first registered directive with a newly defined version with some additional properties (name, scope, restrict and priority are not overwritable)', function () {
       var newLinkFn = angular.noop;
+      var newCompileFn = angular.noop;
```

Comment:
These are all the same fn pointers so better to make new (named) noop functions, otherwise an accidental linkFn/compileFn switch is not detected by the test.